src/config/index.js

```diff
@@ -31,7 +31,9 @@ export const functions = firebase.functions();
 let firebase_messaging;
```

Comment:
@rijusougata13 again :point_left: :smile: ?

Reply:
@shehand  no actually I mentioned in previous comment that I couldn't solve this gitignore  issue in this PR. so I left as it is in this following PR.  But the problem is solved in this new pr #141 . as you can see there is no firebase config file.

Reply:
as it is solved in #141  and it also merged , I am resolving this .